dashboard/static/styles/main.styl

```diff
@@ -13,7 +15,7 @@ main {
 
 section {
     margin-top: 20px;
-    margin-bottom: 20px;
+    margin-bottom: @margin-top;
```

Comment:
We could, definitely. We don't have a need for it just yet, though, since all of our `section`s are within `main`, which is already centered.